src/Resources/contao/dca/tl_point_of_interest.php

```diff
@@ -165,5 +167,23 @@
             'eval' => ['mandatory' => true, 'rgxp' => 'url', 'decodeEntities' => true, 'maxlength' => 255, 'dcaPicker' => true, 'addWizardClass' => false, 'tl_class' => 'w50'],
             'sql' => "varchar(255) NOT NULL default ''",
         ],
+        'addIcon' => [
+            'exclude' => true,
+            'inputType' => 'checkbox',
+            'eval' => ['submitOnChange' => true],
+            'sql' => "char(1) NOT NULL default ''",
+        ],
+        'icon' => [
+            'exclude' => true,
+            'inputType' => 'fileTree',
+            'eval' => [
+                'fieldType' => 'radio',
+                'files' => true,
+                'filesOnly' => true,
+                'extensions' => \Config::get('validImageTypes'),
```

Comment:
Maybe we should use the namespace import `use Contao\Config;` here (and at other occurences)?